lang/cs/Org.Apache.REEF.Client/Org.Apache.REEF.Client.csproj

```diff
@@ -88,7 +98,7 @@ under the License.
     <Compile Include="YARN\RestClient\DataModel\ClusterInfo.cs" />
     <Compile Include="YARN\RestClient\DataModel\ClusterMetrics.cs" />
     <Compile Include="YARN\RestClient\DataModel\Error.cs" />
-    <Compile Include="YARN\RestClient\DataModel\MaximumResourceCapability.cs" />
+    <Compile Include="YARN\RestClient\DataModel\Resouce.cs" />
```

Comment:
From file  comments, it says: "Class generated based on schema provided in...", my understanding is the files names come from scheme. If that is the case, you need to update the name in schema.